test/teos/unit/test_api.py

```diff
@@ -554,3 +555,22 @@ def test_get_appointment_in_responder(internal_api, client, generate_dummy_track
     assert tx_tracker.dispute_txid == r.json.get("appointment").get("dispute_txid")
     assert tx_tracker.penalty_txid == r.json.get("appointment").get("penalty_txid")
     assert tx_tracker.penalty_rawtx == r.json.get("appointment").get("penalty_rawtx")
+
+
+def test_get_subscription_info(internal_api, client, generate_dummy_tracker, monkeypatch):
+    # Need to mock a user
+    internal_api.watcher.gatekeeper.registered_users[user_id].available_slots = 100
```

Comment:
You'll need to register here since otherwise this test won't work standalone. Notice that the current approach won't work given that the user entry for `user_id` does not exist if the other tests are not run.

`internal_api.watcher.gatekeeper.add_update_user(user_id)` should do.

Then if you want to modify the registration data to something custom, feel free to do so.